probcal/samplers/__init__.py

```diff
@@ -0,0 +1,10 @@
+from .gaussian_sampler import GaussianSampler
+from .poisson_sampler import PoissonSampler
+from .nbinom_sampler import NegBinomSampler
+from ..enums import HeadType
```

Comment:
Could we make this an absolute import in case it ever gets moved? So `from probcal.enums import HeadType`